pennylane/ops/op_math/sum.py

```diff
@@ -89,6 +102,11 @@ class Sum(CompositeOp):
         *summands (tuple[~.operation.Operator]): a tuple of operators which will be summed together.
 
     Keyword Args:
+        grouping_type (str): The type of binary relation between Pauli words used to compute
+            the grouping. Can be ``'qwc'``, ``'commuting'``, or ``'anticommuting'``.
+        method (str): The graph coloring heuristic to use in solving minimum clique cover for
+            grouping, which can be ``'lf'`` (Largest First) or ``'rlf'`` (Recursive Largest
+            First).
```

Comment:
Mention that this keyword is ignored when `grouping_type` is `None` (same for other occasions of this description)